ImportError: cannot import name 'Access' from 'access' (unknown location)

The [8 errors in testing](https://github.com/pysal/access/actions/runs/15130991167/job/42531853286#step:5:268) are all flavors of:

```
_________________ ERROR collecting access/tests/test_access.py _________________
ImportError while importing test module '/home/runner/work/access/access/access/tests/test_access.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../../micromamba/envs/access_310/lib/python3.10/site-packages/_pytest/python.py:493: in importtestmodule
    mod = import_path(
../../../micromamba/envs/access_310/lib/python3.10/site-packages/_pytest/pathlib.py:587: in import_path
    importlib.import_module(module_name)
../../../micromamba/envs/access_310/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1050: in _gcd_import
    ???
<frozen importlib._bootstrap>:1027: in _find_and_load
    ???
<frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:688: in _load_unlocked
    ???
../../../micromamba/envs/access_310/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:185: in exec_module
    exec(co, module.__dict__)
access/tests/test_access.py:11: in <module>
    from access import Access, weights
E   ImportError: cannot import name 'Access' from 'access' (unknown location)
```

xref #54

--- a/access/tests/test_access.py
+++ b/access/tests/test_access.py
@@ -1,6 +1,5 @@
-import sys
-
-sys.path.append("../..")
+from access import Access
+from access.access import weights
 
 import math
 import unittest
@@ -8,7 +7,7 @@ import unittest
 import numpy as np
 import pandas as pd
 import geopandas as gpd
-from access import Access, weights
+
 import util as tu
 
 

--- a/access/tests/test_datasets.py
+++ b/access/tests/test_datasets.py
@@ -2,7 +2,7 @@ import unittest
 
 import pandas as pd
 import geopandas as gpd
-from access.datasets import Datasets
+from access import Datasets
 
 
 class TestDatasets(unittest.TestCase):

--- a/access/tests/test_euclidean.py
+++ b/access/tests/test_euclidean.py
@@ -1,6 +1,5 @@
-import sys
-
-sys.path.append("../..")
+from access import Access
+from access.access import weights
 
 import math
 import unittest
@@ -8,7 +7,6 @@ import unittest
 import numpy as np
 import pandas as pd
 import geopandas as gpd
-from access import Access, weights
 import util as tu
 
 

--- a/access/tests/test_floating_catchment_area.py
+++ b/access/tests/test_floating_catchment_area.py
@@ -1,6 +1,5 @@
-import sys
-
-sys.path.append("../..")
+from access import Access
+from access.access import weights
 
 import math
 import unittest
@@ -8,7 +7,6 @@ import unittest
 import numpy as np
 import pandas as pd
 import geopandas as gpd
-from access import Access, weights
 import util as tu
 
 

--- a/access/tests/test_helpers.py
+++ b/access/tests/test_helpers.py
@@ -1,6 +1,5 @@
-import sys
-
-sys.path.append("../..")
+from access import Access
+from access.access import weights, helpers
 
 import math
 import unittest
@@ -8,7 +7,6 @@ import unittest
 import numpy as np
 import pandas as pd
 import geopandas as gpd
-from access import Access, weights, helpers
 import util as tu
 
 

--- a/access/tests/test_hospital_example.py
+++ b/access/tests/test_hospital_example.py
@@ -1,6 +1,5 @@
-import sys
-
-sys.path.append("../..")
+from access import Access
+from access.access import weights
 
 import math
 import unittest

--- a/access/tests/test_misc.py
+++ b/access/tests/test_misc.py
@@ -1,6 +1,5 @@
-import sys
-
-sys.path.append("../..")
+from access import Access
+from access.access import weights
 
 import math
 import unittest
@@ -8,7 +7,6 @@ import unittest
 import numpy as np
 import pandas as pd
 import geopandas as gpd
-from access import Access, weights
 import util as tu
 
 

--- a/access/tests/test_raam.py
+++ b/access/tests/test_raam.py
@@ -1,6 +1,5 @@
-import sys
-
-sys.path.append("../..")
+from access import Access
+from access.access import weights
 
 import math
 import unittest
@@ -8,7 +7,6 @@ import unittest
 import numpy as np
 import pandas as pd
 import geopandas as gpd
-from access import Access, weights
 import util as tu
 
 

--- a/access/tests/test_weighted_catchment.py
+++ b/access/tests/test_weighted_catchment.py
@@ -1,6 +1,5 @@
-import sys
-
-sys.path.append("../..")
+from access import Access
+from access.access import weights
 
 import math
 import unittest
@@ -8,7 +7,6 @@ import unittest
 import numpy as np
 import pandas as pd
 import geopandas as gpd
-from access import Access, weights
 import util as tu
 
 

--- a/access/tests/test_weights.py
+++ b/access/tests/test_weights.py
@@ -1,6 +1,4 @@
-import sys
-
-sys.path.append("../..")
+from access.access import weights
 
 import math
 import unittest
@@ -8,7 +6,6 @@ from random import randint
 
 import numpy as np
 import pandas as pd
-from access import weights
 import util as tu
 
 

